html/webappapis/scripting/event-loops/promise-from-dicarded-ctx.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<head>
+<meta charset=utf-8>
+<link rel=help href="https://html.spec.whatwg.org/multipage/webappapis.html#event-loops">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+  <script>
+// 8.1.4.1 Definitions
+//
+// > [...]
+// >
+// > There must be at least one browsing context event loop per user agent, and
+// > at most one per unit of related similar-origin browsing contexts.
+
+async_test(function(t) {
+  var child = window.open('/common/blank.html');
+  child.onload = function() {
+      var fulfilled = child.Promise.resolve();
+
+      child.close();
+
+      // Create a new Promise (rather than invoke `fulfilled.then` directly) in
+      // order to append to the list stored in the "orphaned" Promise's
+      // [[PromiseFulfilledReactions]] slot.
```

Comment:
Oh, you're right. I mis-read the spec (thinking that the [[PromiseState]] was not set to "fulfilled" synchronously, even for `then`). I'll stop trying to be helpful and just write, "The use of `Promise.resolve` is necessary to trigger incorrect behavior in some browsers at the time of this writing"